spec/features/edit_talk_spec.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Edit Talk', type: :feature do
+  let!(:user) { create_logged_in_user }
+  let!(:talk) { create(:talk, user_id: user.id) }
+
+  context 'when editing a talk with valid params' do
+    before do
+      visit "/talks/#{talk.id}/edit"
+
+      fill_in 'Nome do evento', with: 'RailsConf'
+      click_button 'Salvar'
+    end
+
+    it 'updates the talk' do
+      expect(page).to have_content('Palestra foi atualizado com sucesso.')
+    end
+  end
+
+  context 'when editing a talk with invalid params' do
```

Comment:
would be nice to also having a spec to test the edit of a talk belonging to another user

Reply:
I saw in the request spec, forget this comment